### PR TITLE
feat(Starr:) Add New x266 Custom Formats

### PR DESCRIPTION
**Pull request by yammes08**
# Pull Request

## Purpose

- Resolve: #1836 

## Approach

- [x] Create new x266 custom formats for Radarr and Sonarr
- [x] Add new x266 custom formats to Collection of Custom Formats pages for Radarr and Sonarr
- [x] Build and test locally

## Open Questions and Pre-Merge TODOs

None

## Requirements

- [x] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [x] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: docs/Radarr/Radarr-collection-of-custom-formats.md [hunk 1]
```diff
@@ -1461,6 +1462,24 @@ We've made 3 guides related to this.
 
 ---
 
+### x266
+
+??? question "x266 - [Click to show/hide]"
+
+    Versatile Video Coding (VVC), also known as H.266, ISO/IEC 23090-3, and MPEG-I Part 3, is a video compression standard finalized on 6 July 2020. [Wikipedia](https://en.wikipedia.org/wiki/Versatile_Video_Coding){:target="_blank" rel="noopener noreferrer"}.
+
+    This applies to all x266/VVC releases that are not Remux.
```
Comment (nuxencs) [inline docs/Radarr/Radarr-collection-of-custom-formats.md:1471]: are there even remuxes using `x266`?

File: docs/Sonarr/sonarr-collection-of-custom-formats.md [hunk 2]
```diff
@@ -970,6 +972,24 @@ We've made 3 guides related to this.
 
 ---
 
+### x266
+
+??? question "x266 - [Click to show/hide]"
+
+    Versatile Video Coding (VVC), also known as H.266, ISO/IEC 23090-3, and MPEG-I Part 3, is a video compression standard finalized on 6 July 2020. [Wikipedia](https://en.wikipedia.org/wiki/Versatile_Video_Coding){:target="_blank" rel="noopener noreferrer"}.
+
+    This applies to all x266/VVC releases that are not Remux.
```
Comment (nuxencs) [inline docs/Sonarr/sonarr-collection-of-custom-formats.md:981]: are there even remuxes using `x266`?

File: docs/json/radarr/cf/x266.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          "name": "x|h266",
+          "implementation": "ReleaseTitleSpecification",
+          "negate": false,
+          "required": true,
+          "fields": {
+              "value": "[xh][ ._-]?266|\\bVVC(\\b|\\d)"
+          }
+      },
+      {
+          "name": "Remux",
```
Comment (nuxencs) [inline docs/json/radarr/cf/x266.json:15]:
```suggestion
          "name": "Not Remux",
```
You probably took this from the x265 CF so I guess that one would need changing as well

File: docs/json/radarr/cf/x266.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+  "trash_id": "390455c22a9cac81a738f6cbad705c3c",
+  "name": "x266",
+  "includeCustomFormatWhenRenaming": false,
+  "specifications": [{
```
Comment (nuxencs) [inline docs/json/radarr/cf/x266.json:5]: Formatting nitpick 😅 
You probably took it from the x265 CF so I guess that one would need it as well

**Review by nuxencs — CHANGES_REQUESTED** [review]
Some small stuff, otherwise looks good 👍🏼

File: docs/Radarr/Radarr-collection-of-custom-formats.md (hunk 1 above)
Comment (yammes08) [reply docs/Radarr/Radarr-collection-of-custom-formats.md:1471]: I don't believe so - my assumption was that at the point x/h266 remuxes do exist then we wouldn't want to block them. The CF itself excludes remuxes. Happy to reconsider this though.

File: docs/Sonarr/sonarr-collection-of-custom-formats.md (hunk 2 above)
Comment (yammes08) [reply docs/Sonarr/sonarr-collection-of-custom-formats.md:981]: I don't believe so - my assumption was that at the point x/h266 remuxes do exist then we wouldn't want to block them. The CF itself excludes remuxes. Happy to reconsider this though.

File: docs/json/radarr/cf/x266.json (hunk 3 above)
Comment (yammes08) [reply docs/json/radarr/cf/x266.json:15]: I certainly did! So will change there too.

File: docs/Radarr/Radarr-collection-of-custom-formats.md (hunk 1 above)
Comment (nuxencs) [reply docs/Radarr/Radarr-collection-of-custom-formats.md:1471]: hmm good point, probably no harm in keeping it in

File: docs/Sonarr/sonarr-collection-of-custom-formats.md (hunk 2 above)
Comment (nuxencs) [reply docs/Sonarr/sonarr-collection-of-custom-formats.md:981]: hmm good point, probably no harm in keeping it in

**Review by nuxencs — DISMISSED** [review]
lgtm!

**Review by nuxencs — APPROVED** [review]
👍🏼

**Pull request merged**